src/re_frame/interop.clj

```diff
@@ -0,0 +1,43 @@
+(ns re-frame.interop)
+
+(defn next-tick [f]
+  (.start (Thread. f)))
```

Comment:
I don't like the idea of calling `f` directly because it would mean that in your tests you would see synchronous behaviour that in the real-world would be synchronous.  I think that makes tests a bit less likely to find bugs.  Probably the best thing would be just to use `future` -- the only reason I didn't was I couldn't be bothered double-checking at the time whether it's semantics for exceptions thrown by `f` are sensible for this use-case.  Let me double-check that.